custom_functions/mark_evidence.py

```diff
@@ -1,46 +1,34 @@
-def mark_evidence(container=None, object_id=None, content_type=None, **kwargs):
+def mark_evidence(container=None, input_object=None, content_type=None, **kwargs):
     """
     Mark an object as Evidence in a container
     
     Args:
         container (CEF type: phantom container id): Container ID or Container Object
-        object_id (CEF type: *): Id of object to be added - artifact id, note id, etc.
+        input_object (CEF type: *): The object to mark as evidence. This could be a vault_id, artifact_id, note_id, container_id, or actionrun_id. If the previous playbook block is an action then "keyword_argument:results" can be used for the actionrun_id with the content_type "actionrun". Vault_id can be an ID or a vault hash.
```

Comment:
Now that the content type has been changed from actionrun to actionrun_id, could you please remove `with the content_type "actionrun"` from the end of the third sentence?